lib/kubernetes-deploy/deploy_task.rb

```diff
@@ -541,6 +543,29 @@ def confirm_namespace_exists
       @logger.info("Namespace #{@namespace} found")
     end
 
+    # make sure to never prune the ejson-keys secret
+    def check_applied_ejson_keys_secret
+      raw_secret, st, err = nil
+      with_retries(2) do
+        raw_secret, err, st = kubectl.run("get", "-n", @namespace, "secret",
```

Comment:
The `kubectl` instance injects the namespace for you by default, so no need to include in in the args.